css/index.css

```diff
@@ -1 +1,373 @@
+    color: #b0b0b0;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin: 3.5% 0 0 7%;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 7.8%;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 7.2rem;
+    text-align: center;
+    letter-spacing: .5%;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 9.2% 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 1.6rem;
+    background: white;
+    width: 58%;
+    height: 8%;
+    margin-top: 3.2%;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 9% 0 3% 0;
+   padding: 3.9% 0;
+   font-size: 1.6rem;
+   line-height: 1.8rem;
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 3.6% auto;
+    width: 100%;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 6.5%;
+    padding-top: 5%;
+}
+
+/* Contact Styles */
+
+.contact {
+    padding: 1% 0 0 0;
+}
+
+.contact p, h4 {
+    margin: 1% 0;
+    font-size: 1.6rem;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 4% 0 2%;
+    font-size: 1.6rem;
+}
+
+/* Tablet */
+@media (max-width: 800px) {
+
+    .container {
+        width: 100%;
+    }
+
+    header {
+        flex-wrap: wrap;
+    }
+  
+    header nav {
+        width: 98%;
+        padding: 0 5%;
+    }
+
+    header nav a {
+        text-decoration: none;
+    }
+
+    header nav a:hover {
+        text-decoration: none;
+    }
+
+    a:hover {
+        opacity: 1;
+        transition: 0.2s;
+        border: 5px double black;
+        background-color: darkturquoise;
+        color: black;
+    }
+
+    header .logo {
+        order: -1;
+        display: block;
+        margin: 3.8% auto 0;
+    }
+
+    .cta {
+        justify-content: center;
+    }
+
+    .cta .cta-text button:hover {
+        background: darkturquoise;
+        color: white;
+    }
+    
+    .cta .cta-text {
+        margin: 2% 0 6.8% 0;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       margin: 5% 1.3% 3%;
+    }
+
+    .text-content {
+        margin: 0 1.3%;
+    }
+
+    .contact {
+        margin: 0 3%;
+    }
+}
+
+/* Phone */
+@media (max-width: 500px) {
+
+    header {
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+        flex-direction: column;
+        margin: 0;
+        padding: 0;
+    }    
+
+    @keyframes example {
+        from {background-color: blue;}
+        to {background-color: lightgray;}
+      }
+
```

Comment:
Nice animation! I'm glad you went for a stretch goal. I would encourage you to keep digging into animation. People tend to be really impressed by even simple animation. It's a nice skill to have under your belt to show clients that you have "magic powers."